code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -157,6 +175,24 @@
 	icon_state = "orangews"
 	gas_type = GAS_HYDROGEN
 
+/obj/machinery/portable_atmospherics/canister/methane
+	name = "methane canister"
+	desc = "Methane. Used in thruster fuel along with kitchen stoves."
+	icon_state = "methane"
+	gas_type = GAS_METHANE
+
+/obj/machinery/portable_atmospherics/canister/ammonia
+	name = "ammonia canister"
+	desc = "Ammonia. Used in industrila processes and is also very poor for your eyes and lungs."
```

Comment:
Industrial.

I'd cut out the warnings about eyes and lungs (and other warnings on canisters prior). If we want to be creative we could make a small chemical warning sprite and have examine_closely display the assorted warnings the can has.